dashboard/dashapi/dashapi.go

```diff
@@ -114,6 +114,8 @@ type Commit struct {
 	BugIDs     []string // ID's extracted from Reported-by tags
 	Date       time.Time
 	Link       string // set if the commit is a part of a reply
+	Repo       string
```

Comment:
I think we shouldn't change API structs to pass internal data inside of a single app. Nobody is sending/receiving these on the dashboard API level.

Reply:
Let's recover uiCommit then.
We have 2 types to represent commit data and makeVCSCommit() function makes no sense (I don't see readable alternative to support 2 input types).
2 vcsCommit{} inline inits are needed to support 2 different type conversions.
PTAL.